mmdet/models/detectors/two_stage.py

```diff
@@ -106,25 +88,10 @@ def forward_dummy(self, img):
         if self.with_rpn:
             rpn_outs = self.rpn_head(x)
             outs = outs + (rpn_outs, )
-        proposals = torch.randn(1000, 4).to(device=img.device)
-        # bbox head
-        rois = bbox2roi([proposals])
-        if self.with_bbox:
-            bbox_feats = self.bbox_roi_extractor(
-                x[:self.bbox_roi_extractor.num_inputs], rois)
-            if self.with_shared_head:
-                bbox_feats = self.shared_head(bbox_feats)
-            cls_score, bbox_pred = self.bbox_head(bbox_feats)
-            outs = outs + (cls_score, bbox_pred)
-        # mask head
-        if self.with_mask:
-            mask_rois = rois[:100]
-            mask_feats = self.mask_roi_extractor(
-                x[:self.mask_roi_extractor.num_inputs], mask_rois)
-            if self.with_shared_head:
-                mask_feats = self.shared_head(mask_feats)
-            mask_pred = self.mask_head(mask_feats)
-            outs = outs + (mask_pred, )
+        proposals = torch.randn(1000, 4).cuda()
```

Comment:
It is better to be on the same device as `img`.

Reply:
Sure